homeassistant/components/bond/const.py

```diff
@@ -10,3 +10,9 @@
 HUB = "hub"
 BPUP_SUBS = "bpup_subs"
 BPUP_STOP = "bpup_stop"
+
+SERVICE_SET_FAN_SPEED_BELIEF = "set_fan_speed_belief"
```

Comment:
Is `belief` a device native concept that we need to keep. Otherwise I think it's not a good name. I'd name this `set_assumed_fan_speed`.

Reply:
Bond actually calls this state belief in the api docs: http://docs-local.appbond.com/#tag/Device-State/paths/~1v2~1devices~1%7Bdevice_id%7D~1state/patch


The bond app calls it tracked state.  I think that is actually a better description of what's going on here and would probably be the most easily understood by someone who has used the bond app to setup the device (I don't think there is another reasonable way to assume it would be done)

Reply:
Sounds good.

Reply:
@joshs85 Are you able to make the requested changes?

Reply:
Change made in https://github.com/home-assistant/core/pull/56147/